click_odoo_contrib/update.py

```diff
@@ -223,7 +224,14 @@ def _update_db_nolock(
     watcher=None,
     list_only=False,
     ignore_addons=None,
+    only_compute_hashes=False,
 ):
+    if only_compute_hashes:
+        with conn.cursor() as cr:
+            _save_installed_checksums(cr, ignore_addons)
+        _logger.info("Only compute module hashes, update is not performed.")
+        return
```

Comment:
DRY: to avoid having two calls to _save_installed_checksums in different circumstances, we could may be call _save_installed_checksums in `_update_db` ?